apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -923,7 +923,133 @@
                     "default" : "ae_mainnet"
                 }
             }
+        },
+        "stratum": {
+            "type"    : "object",
+            "additionalProperties": false,
+            "properties" : {
+                "enabled" : {
+                    "type" : "boolean",
+                    "default" : false
+                },
+                "connection": {
+                    "type"    : "object",
+                    "additionalProperties": false,
+                    "properties" : {
+                        "host" : {
+                            "type" : "string",
+                            "default" : "pool.aeternity.com"
+                        },
+                        "port" : {
```

Comment:
Could you please propose integration of doc re API exposed by node? I mean a further bullet point at the end of [this list](https://github.com/aeternity/protocol/blame/3d7c855bce7f6fc3331cb1517cf4406e001fd7f2/node/api/README.md#L17) i.e. at same level of "External HTTP endpoint" or "Internal HTTP endpoint", maybe [at the end](https://github.com/aeternity/protocol/blame/3d7c855bce7f6fc3331cb1517cf4406e001fd7f2/node/api/README.md#L40).

Reply:
Added to todo list.